proto/minder/v1/minder.proto

```diff
@@ -452,6 +452,42 @@ service UserService {
         };
     }
 
+    // ListInvitations returns a list of invitations for the user
+    // based on the user's registered email address.  Note that a
+    // user who receives an invitation nonce may still accept the
+    // invitation even if the nonce was directed to a different
+    // email address.  This is beacuse understanding the routing of
+    // email messages is beyond the scope of Minder.
+    //
+    // This API endpoint may be called without the logged-in user
+    // previously having called `CreateUser`.
+    rpc ListInvitations (ListInvitationsRequest) returns (ListInvitationsResponse) {
+        option (google.api.http) = {
+            get: "/api/v1/user/invitations"
+        };
+
+        option (rpc_options) = {
+            target_resource: TARGET_RESOURCE_USER
```

Comment:
Can you remind me what is the default `relation`? I see that we don't specify one with the other methods in the same service either.

Reply:
The default 0 value is `TARGET_RESOURCE_UNSPECIFIED`.  The other options are `TARGET_RESOURCE_NONE` and `TARGET_RESOURCE_PROJECT`.

Reply:
IIRC, except for `TARGET_RESOURCE_PROJECT`, all the other options currently use the same Authz code.  Additionally, all of these requests operate on an implicit subject of the logged in user, rather than operating on a project, provider, repository, etc.